Simplified/Settings/NYPLSettingsDeleteLibraryCardViewController.swift

```diff
@@ -0,0 +1,205 @@
+  }
+  
+  @objc private func dismissVC() {
+    self.navigationController?.dismiss(animated: true, completion: nil)
+  }
+  
+  // MARK: - Setup
+  
+  private func setupNavBar() {
+    self.title = NSLocalizedString("Delete Library Card",
+                                   comment: "Title for navigation bar")
+    
+    let backButton = UIBarButtonItem(title: NSLocalizedString("Cancel",
+                                                              comment: "Button title for dismissing view controller"),
+                                     style: .plain,
+                                     target: self,
+                                     action: #selector(dismissVC))
+    
+    self.navigationItem.leftBarButtonItem = backButton
+  }
+  
+  private func setupUI() {
+    self.tableView.backgroundColor = NYPLConfiguration.primaryBackgroundColor
+    self.tableView.separatorStyle = .none
+    self.tableView.allowsSelection = false
+  }
+  
+  // MARK: - Helper
+  
+  lazy var footerView: UIView = {
+    let view = UIView()
+    view.backgroundColor = NYPLConfiguration.primaryBackgroundColor
+    return view
+  }()
+  
+  lazy var emailButton: UIButton = {
+    let button = UIButton()
+    button.addTarget(self, action: #selector(presentEmailComposer), for: .touchUpInside)
+    return button
+  }()
+  
+  private func libraryCardDeletionDescription() -> NSMutableAttributedString {
+    let paragraphStyle = NSMutableParagraphStyle()
+    paragraphStyle.lineSpacing = 1.25
+    paragraphStyle.lineHeightMultiple = 1.25
+    
+    let description = NSLocalizedString("DeleteLibraryCardDescription",
+                                        comment: "Description of the delete action")
+    let attributedString = NSMutableAttributedString(string:description,
+                                                     attributes: [.font: UIFont.systemFont(ofSize: 14),
+                                                                  .foregroundColor: NYPLConfiguration.primaryTextColor,
+                                                                  .paragraphStyle: paragraphStyle]
+    )
+
+    return attributedString
+  }
+}
+
+extension NYPLSettingsDeleteLibraryCardViewController: MFMailComposeViewControllerDelegate {
+  func mailComposeController(_ controller: MFMailComposeViewController, didFinishWith result: MFMailComposeResult, error: Error?) {
+    controller.dismiss(animated: true, completion: nil)
+    
+    switch result {
+    case .failed:
+      if let error = error {
+        let alert = NYPLAlertUtils.alert(title: "Error", error: error as NSError)
+        NYPLAlertUtils.presentFromViewControllerOrNil(alertController: alert,
+                                                      viewController: self,
+                                                      animated: true,
+                                                      completion: nil)
+      }
+    case .sent:
+      // TODO: Check wordings with Risa
```

Comment:
@ettore The ticket/figma does not describe the behaviour after email is sent, so I am displaying an alert and a dismiss action here. Will check with Andrew/Risa.

Also, this PR contains commits that are for the iOS-411 ticket and those are reviewed. Only the last commit (762441f23a2) needs to be reviewed for this PR.

Reply:
looks good to me, so does this mean you will just merge this PR instead of the other one?

Reply:
Yeah I think merging this PR instead of both PRs make more sense

Reply:
Turns out github automatically merged [the other PR](https://github.com/NYPL-Simplified/Simplified-iOS/pull/1569) when I merge this PR, because this PR contains the commits from the other PR